src/picknik_ur_multi_arm_config/config/moveit/multi_arm_ur.srdf

```diff
@@ -182,27 +182,6 @@
     <disable_collisions link1="first_forearm_link" link2="first_upper_arm_link" reason="Adjacent"/>
     <disable_collisions link1="first_forearm_link" link2="first_wrist_1_link" reason="Adjacent"/>
     <disable_collisions link1="first_forearm_link" link2="first_wrist_2_link" reason="Never"/>
-    <disable_collisions link1="first_forearm_pinch_link" link2="first_forearm_link" reason="Adjacent"/>
-    <disable_collisions link1="first_forearm_pinch_link" link2="first_tool_changer_link" reason="Never"/>
-    <disable_collisions link1="first_forearm_pinch_link" link2="first_upper_arm_link" reason="Never"/>
-    <disable_collisions link1="first_forearm_pinch_link" link2="first_wrist_1_link" reason="Never"/>
-    <disable_collisions link1="first_forearm_pinch_link" link2="first_wrist_2_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_realsense_camera_adapter_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_base_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_left_finger_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_left_finger_tip_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_left_inner_knuckle_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_left_knuckle_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_right_finger_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_right_finger_tip_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_right_inner_knuckle_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_robotiq_85_right_knuckle_link" reason="Never"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_tool_changer_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_ur_to_robotiq_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_wrist_1_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_wrist_2_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_wrist_3_link" reason="Adjacent"/>
-    <disable_collisions link1="first_wrist_3_pinch_link" link2="first_wrist_mounted_camera_link" reason="Never"/>
```

Comment:
Are we sure we want to remove the pinch link SRDF collision exceptions? If anything we should probably enable the pinch link on the multi arm xacro and then reenable these. Thoughts @sjahr?

I'm not necessarily blocking this PR on that though.